Gems/Atom/RHI/DX12/Code/Source/RHI/StreamingImagePool.h

```diff
@@ -65,10 +68,26 @@ namespace AZ
             // Packed mips occupy a dedicated set of tiles.
             void AllocatePackedImageTiles(Image& image);
 
-            RHI::Ptr<ID3D12Heap> m_heap;
+            // Get the data reference of device heap memory usage 
+            RHI::HeapMemoryUsage& GetDeviceHeapMemoryUsage();
+
+            // whether to enable tiled resource
+            bool m_enableTileResource = false;
 
+            // mutex to protect tile allocation and de-allocation from any threads
             AZStd::mutex m_tileMutex;
-            RHI::PoolAllocator m_tileAllocator;
+
+            // for allocate heap page
+            HeapAllocator m_heapPageAllocator;
+
+            // for allocate tile from heap page
+            TileAllocator m_tileAllocator;
+
+            // The default tile for null tiles.
```

Comment:
This sentence needs to be better phrased. Will this work?
The default tile is initialized in order to support images that have no tiles.

Reply:
This is not fully tested. I may update the code when I have example setup for this case.